fv_be/app/models/part_of_speech.py

```diff
@@ -0,0 +1,23 @@
+from django.db import models
+
+# FirstVoices
+from .base import BaseModel
+
+
+class PartOfSpeech(BaseModel):
```

Comment:
Would there be any reason for the parts of speech model to have a foreign key linking to the site the category belongs to?

Reply:
Parts of speech will be an independent list as of now, so it will not require a foreign key to the site as of now.
Future plans are for each site to maintain their own list for parts of speech but they are not implemented as of now.